sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaIO.java

```diff
@@ -1049,6 +1051,27 @@ private void nextBatch() {
       curBatch = Iterators.cycle(nonEmpty);
     }
 
+    private void setupInitialOffset(PartitionState p) {
```

Comment:
nit: `p` is not a good variable name, and we actually use it a lot for `Pipeline`